src/bindings.rs

```diff
@@ -20,6 +22,83 @@ const TAG_UNDEFINED: i64 = 3;
 const TAG_EXCEPTION: i64 = 6;
 const TAG_FLOAT64: i64 = 7;
 
+#[cfg(feature = "bignum")]
+#[derive(Clone, Debug)]
+pub enum BigIntOrI64 {
```

Comment:
Let's move those types and impls into `value/bigint.rs`. 

(`#[cfg]` attr goes on the `mod bigint` declaration)

Reply:
I didn't put it there initially because to do it the `inner` field should be made public or more wrapping code added to allow inspection in tests.